js/lib/DateFmt.js

```diff
@@ -852,13 +852,11 @@ DateFmt.prototype = {
 
     // stand-alone of m (month) is l
     // stand-alone of my (month year) is mys
-    // stand-alone of d (day) is a
     // stand-alone of w (weekday) is e
     // stand-alone of y (year) is r
     _standAlones: {
         "m": "l",
         "my": "mys",
-        "d": "a",
```

Comment:
oh.. Sorry for the confusion.;( We need to keep day's standAlone data. There are related test cases. such as Japanese, Korean.. we need to keep that info, probably  extract from cldr. `availableFormats-> d`?